drivers/flash/flash_mcux_flexspi_nor.c

```diff
@@ -1194,6 +1194,39 @@ static int flash_flexspi_nor_check_jedec(struct flash_flexspi_nor_data *data,
 		/* Device uses bit 1 of status reg 2 for QE */
 		return flash_flexspi_nor_quad_enable(data, flexspi_lut,
 						     JESD216_DW15_QER_VAL_S2B1v5);
+	case 0x1740EF:
+		/* W25Q64JV flash, use 3 byte read/write */
+		flexspi_lut[READ][0] = FLEXSPI_LUT_SEQ(
+			kFLEXSPI_Command_SDR, kFLEXSPI_1PAD, SPI_NOR_CMD_4READ,
+				kFLEXSPI_Command_RADDR_SDR, kFLEXSPI_4PAD, 24);
+		/* Flash needs 6 dummy cycles */
```

Comment:
Are we sure about this? I looked at the W25W64JV datasheet and it documents eight dummy cycles for various read operations.